### ISSUE TITLE
Stretchy elements should inherit current color

### ISSUE THREAD

**Issue by glebm** [Khan/KaTeX#795]
If no color is set explicitly, stretchy elements should still have a mask image with `currentColor`:

```js
node.style.backgroundColor = 'currentColor';
```

Otherwise this happens:

```
$$
\overbrace{x+y}
$$
```

![overbrace](https://user-images.githubusercontent.com/216339/29335184-b67def6a-8201-11e7-8d60-836af456e3e6.png)

@ronkok

**Comment by edemaine** [comment]
I was also wondering whether any background settings should be marked `!important`.  According to [this discussion](https://stackoverflow.com/questions/3893986/css-media-print-issues-with-background-color), it seems to help with printing on most browsers...

**Comment by ronkok** [comment]
Sometimes I wish that I had written all of the stretchy element SVG code inline. With what I know today, I no longer have to write it with URL escapes. With just JavaScript escapes, the code is somehow less hideous to look upon.  Colors are easy with inline code. The CSS becomes very straightforward.

Here is the [SVG inline code](https://github.com/ronkok/KaTeX/blob/master/src/delimiter.js#L316-L357) for six different surd characters. There are 36 external SVG files currently in the images folder. Transferring it all to inline would add about 20 KB to the JavaScript.

Any takers?

**Comment by ronkok** [comment]
Oh, and those inline SVG are not background, they are instead `<svg>` written directly into the DOM. All the background printing issues go away.

**Comment by kevinbarabash** [comment]
Another benefit of the SVG being inline is that if you serve it gzip'd a lot of the duplication will get compressed away.

**Comment by ronkok** [comment]
I expect that this issue is now resolved, since stretchy elements are all inline.

**Comment by kevinbarabash** [comment]
@glebm this should be fixed with the release of (v0.8.3)[https://github.com/Khan/KaTeX/releases/tag/v0.8.3].  Please reopen if the new version doesn't solve this issue.

**Comment by glebm** [comment]
Confirmed, it works! Thank you!
